tests/TestUTestMacro.cpp

```diff
@@ -421,6 +421,28 @@ static int _countingMethod()
     return countInCountingMethod++;
 }
 
+TEST(UnitTestMacros, LONGS_EQUAL_macroExpressionSafety)
+{
+    // If the expansion of arguments to LONGS_EQUAL() are not parenthesized,
```

Comment:
Could you remove the comments, and instead ensure that the test case name describes the case so that no additional comments are needed?